src/lib.rs

```diff
@@ -1150,7 +1150,7 @@ unsafe fn close_internal(c_out: &mut *mut Rav1dContext, flush: c_int) {
         rav1d_free_aligned((*f).ts as *mut c_void);
         rav1d_free_aligned((*f).ipred_edge[0] as *mut c_void);
         free((*f).a as *mut c_void);
-        free((*f).tile as *mut c_void);
+        (*f).tiles = Default::default();
```

Comment:
We should remove this when context is owned and dropped properly, let's mark this with a TODO.

Reply:
Done in 03c4d28.